android/app/build.gradle

```diff
@@ -101,9 +101,12 @@ android {
         ndk {
             abiFilters "armeabi-v7a", "x86"
         }
-        manifestPlaceholders = [manifestApplicationId: "${applicationId}",
-                        onesignal_app_id: "aa46a500-ab1c-4127-b9ff-e7373da3ce35",
-                        onesignal_google_project_number: "185558680648"]
+        manifestPlaceholders = [
+            manifestApplicationId: "${applicationId}",
+            onesignal_app_id: "aa46a500-ab1c-4127-b9ff-e7373da3ce35",
+            onesignal_google_project_number: "185558680648",
+            gmaps_key: System.getenv("GOOGLE_MAPS_API_KEY") ?: "",
```

Comment:
I made the change so that it would work locally (non-travis). It most likely needs to be the same name for it to work on both travis and local development.